config/crd/kustomization.yaml

```diff
@@ -8,4 +8,4 @@ resources:
 - bases/numaplane.numaproj.io_monovertexrollouts.yaml
 - bases/numaplane.numaproj.io_numaflowcontrollers.yaml
 # install numaflow minimal CRDs as a dependency
-- https://github.com/numaproj/numaflow/config/advanced-install/minimal-crds?ref=stable
+- https://github.com/numaproj/numaflow/config/advanced-install/minimal-crds?ref=v1.5.2
```

Comment:
I guess we'll need to make sure we update this going forward as well.

On one hand, we could try "latest", but I wonder if that may cause its own failures.